config/filter.d/cyrus-imap.conf

```diff
@@ -13,7 +13,7 @@ before = common.conf
 
 _daemon = (?:cyrus/)?(?:imap(d|s)?|pop3(d|s)?)
 
-failregex = ^%(__prefix_line)sbadlogin: \S+ ?\[<HOST>\] \S+ .*?\[?SASL\(-13\): (authentication failure|user not found): .*\]?$
+failregex = ^%(__prefix_line)sbadlogin: (\S+)? ?\[<HOST>\] \S+ .*?\[?SASL\(-13\): (authentication failure|user not found): .*\]?$
```

Comment:
1.  I know that this one is non-capturing, but does it give us any advantage over a more concise capturing `(...)`?
2. I know that it is optional ATM, but it imho should be made non optional if leading `\[` is present